packages/xo-server/src/api/schedule.mjs

```diff
@@ -64,3 +66,26 @@ delete_.params = {
 }
 
 export { delete_ as delete }
+
+export async function runSequence({ idSchedules }) {
+  const t = new Task({ properties: { type: 'xo:schedule:sequence', name: 'Schedule sequence' } })
```

Comment:
I think we should make it a top level task, it will be easier to display its logs and handle abortion:

```suggestion
  const t = this.tasks.create({ type: 'xo:schedule:sequence', name: 'Schedule sequence' })
```

What do you think?